.github/workflows/builds.yml

```diff
@@ -13,27 +13,27 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        python: [3.7, 3.8]
+        python: ["3.8", "3.10"]
```

Comment:
I suggest testing consecutive versions instead. I see two reasons behind this:
- for another project, I recently took a look at the most used python versions, and 3.8 was the top one. Adoption is generally smooth, meaning there are way more people using 3.9 than 3.10.
- once we drop support of 3.8, it would be nice to already have a history of test runs on 3.9